src/pages/services/aem-cf-console-admin/api/action-bar/index.md

```diff
@@ -90,6 +91,29 @@ const guestConnection = await register({
 });
 ```
 
+## Custom button with sub menu
+
+```js
+import { register } from "@adobe/uix-guest";
+
+// ...
+
+const guestConnection = await register({
+    id: "my.company.extension-with-action-bar-button",
+    methods: {
+        actionBar: {
+            deleteButtons() {
+                return [
+                    {
+                        id: "my.company.export-button"
```

Comment:
Please describe how a developer can find out ID of the button or list available IDs with button descriptions.
As well, please ensure a signed CLA

Reply:
It is described in Delete button API, but I will add additional comment here